src/operator/tensor/la_op_inline.h

```diff
@@ -403,15 +417,18 @@ struct potrf_backward {
     // Here, copyLTU(M) creates a symmetric matrix from the square matrix M
     // by setting the upper triangle to be equal to the lower triangle, leaving
     // lower triangle and diagonal unchanged.
+    // The function also handles the case when L is upper triangular by appropriate
+    // transpositions.
```

Comment:
Can you change the variable names here to be consistent with the new doc? change dL and L to dB and B both in the comment and in the op(args) for the function.

Reply:
Will change L/dL for potrf/potri to B/dB.  Though this will trigger a bigger renaming as "L" for lower triangular is also used in trsm/trmm and this would be then inconsistent too. So for the latter functions, I did a remapping of B-->C, A-->B, L-->A which actually brings this internal naming in line with the external documentation (having "C" denoting "out").